lib/stale.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Determines if an issue or PR is stale and create a message if needed.
+ * Will only process when the configuration for days considered stale
+ * is set for Pulls and Issues respectively.
+ *
+ * Assumes that the numbers of days are always set if stale property in Settings
+ * is defined.
+ *
+ * @return
+ *  JSON object with the properties mergeable and the description if
+ *  `mergeable` is false
+ */
+module.exports = async (context, settings) => {
```

Comment:
this function seems redundant, i think you can just do `module.exports = process`

Reply:
It is not redundant. I have to return the promise so that if it is rejected jest will fail the test.